src/components/PopupWithForm.js

```diff
@@ -7,6 +7,14 @@ export default class PopupWithForm extends Popup {
     this._handleFormSubmit = handleFormSubmit;
   }
 
+  setLoading(isLoading, submitSave) {
+    if (isLoading) {
```

Comment:
You can save the initial text in the constructor instead of passing it via arguments

```js
constructor() {
  ..
  this._buttonText = this._button.textContent;
```